atd-vze/src/views/VZDashboard/VZDashboard.js

```diff
@@ -5,41 +5,31 @@ import { useQuery } from "@apollo/react-hooks";
 import Widget02 from "../Widgets/Widget02";
 import VZLinksWidget from "../Widgets/VZLinksWidget";
 import VZNoticeWidget from "../Widgets/VZNoticeWidget";
-import { format, subDays } from "date-fns";
+import { subDays } from "date-fns";
 
 import { GET_CRASHES_YTD } from "../../queries/dashboard";
 
 import bi_logo from "../../assets/img/brand/power_bi_icon_white_on_transparent.png";
 
 function VZDashboard() {
   const year = new Date().getFullYear();
-  const yearStart = `${year}-01-01`;
+  const yearStart = `${year}-01-01T00:00:00`;
```

Comment:
little TZ bug here—to get the `year` value you want to use `new Date().getUTCFullYear()` since you're constructing a UTC timestamp